spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'vanagon/platform'
+load File.expand_path(File.join(File.dirname(__FILE__), "../../../..", "lib", "vanagon.rb"))
```

Comment:
You shouldn't need to do that here. It happens there because vanagon isn't loaded yet. Vanagon is loaded already here.